### PR TITLE
Adds internal Bleeding and other medical tweaks and nerfs

### PR DESCRIPTION
**Pull request by Jamini**
## About The Pull Request

Adds internal bleeding. Damaged blood vessel organs now will cause you to lose blood and can be lethal!
Damaged blood vessels show on examine text
image

Quickclot (Clonedextrone + Kelotane) can repair blood vessels internally without surgery. Bicaridine and Tricordrazine can slow internal bleeding. Blood injections, Iron suppliments, or an IV can help keep someone stable for surgery.

Emergency autoinjectors now come with 1u quickclot in them. This will heal about 5 damage to your blood vessels.

* Minor Arterial/Vein tears (below 5 damage) will autoheal slowly. REsulting in at max 10% of total blood volume lost.
* Moderate Arerial/Vein tears (6-8 damage) will take about fifteen minutes to kill you.
* Major Arterial/Vein tears (13+ damage) will take about ten
* The rate at which you bleed slows as you lose blood.

Nerves, Muscles, and Blood vessels now show where they are in your body on the advanced scanner
image

Manually spawned humans no longer show up as Tar/voidpeople.

Organs now cost 3 biomatter instead of 20. Bringing corpses and monkies in-line with the clone vats.
The Mekhane now is reduced to a single keg (in storage) of Cahors.
More Cahors can be printed at the bioprinter for 10 biomatter sheets (100 biomass) and 1 glass ( the bottle)

## Why It's Good For The Game

The first in a number of planned medical nerfs and tweaks.
More functionality for erismed organs
More is planned/on the way.

## Changelog
```changelog
add: Internal Bleeding
tweak: The Mekhane can now print Cahor bottles
tweak: Quickclot now heals damaged blood vessels
tweak: Internal Bleeding shows on examine text
tweak: Emergency autoinjectors (the one in your box) now have 1u quickclot.
fix: Tarpeople have been given a hot shower.
fix: Organic organs now have a sand biomatter cost
map: Mekhane Cahor stocks have been drastically reduced
map: The Mekhane now spawn with two empty BIDON canisters along with their keg.
```

### CODE REVIEW

File: zzzz_modular_occulus/code/modules/stashes/stashes.dm [hunk 1]
```diff
@@ -90,7 +90,7 @@
 	Is the tale of Jayme Dawson and his crew <br>\
 	Yes, the tale of Dawson's Christian and her crew.<br>\
 	<br>\
-	D%"
+	%D"
```
Comment (Michiyamenotehifunana) [inline zzzz_modular_occulus/code/modules/stashes/stashes.dm:93]: Redundant typo fix already present on an already-merged PR.

**Review by Michiyamenotehifunana — CHANGES_REQUESTED** [review]
Nothing looks out of the ordinary aside from the redundant stash typo fix.

**Comment by zackerycodestoo** [comment]
I will also ask that unrelated commits be split into separate pulls -- it's nice to see many bugfixes, but tarpeople/less cahors will also be reverted if there are issues with this code that need reverting

**Review by Michiyamenotehifunana — APPROVED** [review]
Looks about right despite being unatomic. All good.

**Comment by Jamini** [comment]
> I will also ask that unrelated commits be split into separate pulls for organizational purposes -- it's nice to see many bugfixes, but tarpeople/less cahors will also be reverted if there are issues with this code that need reverting

Normally I would do so, however most of these fixes are things that I fixed while I was working on the main pull. Either things I spotted or related issues.

**Pull request merged**